app/controllers/users_controller.rb

```diff
@@ -1,14 +1,14 @@
 class UsersController < ApplicationController
-  def new
-    @user = User.new
-  end
+  before_action :load_user, only: %i(show edit update)
+  before_action :logged_in_user, only: %i(edit update index destroy)
```

Comment:
```suggestion
  before_action :logged_in_user, only: %i(edit update index destroy)
  before_action :load_user, only: %i(show edit update)
```
ri chứ em, check xem user đã login chưa mới load user chứ

2.  only: %i(edit update index destroy)
quá nửa actions ( 7 actions) -> tìm hiểu `except`  đi nhé